packages/payment-detection/src/erc777/superfluid-detector.ts

```diff
@@ -21,15 +22,65 @@ export class SuperFluidPaymentDetector extends ReferenceBasedDetector<
     super(PaymentTypes.PAYMENT_NETWORK_ID.ERC777_STREAM, advancedLogic.extensions.erc777Stream);
   }
 
+  protected isSubsequentRequest(request: RequestLogicTypes.IRequest): boolean {
+    return !!request.extensions[ExtensionTypes.ID.PAYMENT_NETWORK_ERC777_STREAM].values
+      .masterRequestId;
+  }
+
   /**
-   * Extracts the balance and events of an address
+   * This returns the specific values we store for the ERC777 extensions.
+   * The extra fields specific for this extension are:
+   * expectedStartDate: The expected start date of streaming
+   * expectedFlowRate: extensionAction.parameters.expectedFlowRate,
+   * previousRequestId: The previous requestId in the streaming series
+   * masterRequestId: The first requestId of the streaming series
+   * recurrenceNumber: The position of this request in the streaming series, 0 indexed.
```

Comment:
why not have a type?

Reply:
Ah yes, Leo added this later, I'll include it from types